### PR TITLE
8 tool functions are built separately from docker launch

### PR DESCRIPTION
**Pull request by barnettwilliam**
For #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,23 +36,22 @@ git clone --recurse-submodules git@github.com:mdenet/educationplatform-docker.gi
...
 
-## Build and run eclipse-based tool services
-
-This builds the eclipse-based tool services e.g. OCL using maven and starts the services.
-```
-cd ./educationplatform-docker/platformtools/services
-mvn clean 
-mvn install
-./launch.sh
+docker build ./platformtools -t mdenet-tool-build:latest -f platformtools/services/Dockerfile  && \
```
Comment (szschaler) [inline README.md:40]: Can we get rid of this need to pre-build? Each tool's Dockerfile should copy in exactly the java code it needs and then do its own `mvn clean`, `mvn install`.

**Review by szschaler — CHANGES_REQUESTED** [review]
Improve architectural clarity by removing the shared image and building separate images for each tool.

File: README.md (hunk 1 above)
Comment (barnettwilliam) [reply README.md]: Yes though it requires more disk space and initial build time approx 20GB vs 12GB and 15min vs 10min as the tool builds are duplicated.

**Comment by barnettwilliam** [comment]
Made this change in 87f52b5e2b8356d4844c9ceabed367331d26dda8

> Improve architectural clarity by removing the shared image and building separate images for each tool.

File: README.md [hunk 2]
```diff
@@ -37,23 +36,21 @@ git clone --recurse-submodules git@github.com:mdenet/educationplatform-docker.gi
...
-cd ./educationplatform-docker/platformtools/services
-mvn clean 
-mvn install
-./launch.sh
-```
-> Note that this can take some time to complete the first time the services are built.
+## Access the platform
 
-Further details of launching a tool service manually can be found in the platformtools [readme](https://github.com/mdenet/platformtools). 
+Once docker has [started](#build-and-run-the-docker-image) the containers, the platform will be available at http://127.0.0.1:8080
```
Comment (szschaler) [inline README.md:48]: Worth adding that users will need to use `Ctrl-C` in the terminal running `docker compose up` to stop the platform safely?

**Review by szschaler — APPROVED** [review]
One minor comment on the documentation, but otherwise looks good to go.

**Pull request merged**